src/Adapter/MSTest.TestAdapter/Discovery/AssemblyEnumerator.cs

```diff
@@ -300,8 +303,13 @@ private static bool DynamicDataAttached(UnitTestElement test, Lazy<TestMethodInf
         return testMethodInfo.Value != null && TryProcessTestDataSourceTests(test, testMethodInfo.Value, tests);
     }
 
+    [UnconditionalSuppressMessage("Aot", "IL3000:DoNotUseLocation", Justification = "Fixture tests are not supported in NativeAOT mode.")]
+
     private static void AddFixtureTests(TestMethodInfo testMethodInfo, List<UnitTestElement> tests, HashSet<string> fixtureTests)
     {
+#if NET8_0_OR_GREATER
```

Comment:
Could be `NETCOREAPP` or `NET7_0_OR_GREATER` if we want to link it specifically to native aot.